src/value-indexer/src/test/scala/VerifyValueIndexer.scala

```diff
@@ -0,0 +1,88 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
```

Comment:
Need to add tests for missing values in levels (make sure they are at the end), floating point levels (where x != y but x.toString == y.toString, infs and nans in the column etc)